CONTRIBUTING.md

```diff
@@ -154,6 +155,15 @@ Names of project files should:
 Mello uses the BEM Naming structure, which aims to keep CSS codebases structured with a consistent naming convention.
 _For more information, see https://yandex.com/dev/bem/_
 
+### Dark and Light Mode
+
+The default colour scheme for the application is dark mode, and as such all components should have their default stylign consistent with dark mode. However, the application also has support for light mode, and your component/extensions must also have support for light mode. The following steps will add light mode support for your component:
```

Comment:
Just noticing something here - when I checkout `main-group2` and run the application, it defaults to light mode even though I have not touched any settings. Is this a bug I am experiencing, or is light mode the new default?

Reply:
The default is whatever your system is in (i.e. based on if your Mac/pc is in dark/light mode)

Reply:
ah okay - would it make more sense to say this then?